### PR TITLE
Fix: Incorrect clip region issue.

### PR DESCRIPTION
**Pull request by H1Gdev**
### Sample SVG
```svg
<?xml version="1.0" encoding="UTF-8"?>
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="200" height="200">
<defs>
<clipPath id="clipper">
<!-- specified stroke-width to clarify this issue. -->
<rect x="50" y="50" width="100" height="100" stroke-width="80" />
</clipPath>
</defs>
<circle cx="100" cy="100" r="100" fill="red" clip-path="url(#clipper)" />
</svg>
```

### Expected
expected

### Actual
actual

### CODE REVIEW

**Comment by H1Gdev** [comment]
I found this issue when fixing #479.

**Comment by H1Gdev** [comment]
This PR fixes issue #363.

File: Tests/W3CTestSuite/svg/__pull_request-492-01.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mrbean-bremen) [inline Tests/W3CTestSuite/svg/__pull_request-492-01.svg]: Very nice :) Could you also add this to the passing tests (both the csv and the txt files, I may unify these later...)?

**Comment by H1Gdev** [comment]
@mrbean-bremen 

Thank you for your review.

I add these SVGs to the passing tests!

**Pull request merged**